src/main/kotlin/no/nav/familie/tilbake/behandling/steg/Foreslåvedtakssteg.kt

```diff
@@ -47,6 +55,18 @@ class Foreslåvedtakssteg(val behandlingskontrollService: BehandlingskontrollSer
         behandlingskontrollService.fortsettBehandling(behandlingId)
     }
 
+    private fun håndterOppgave(behandlingId: UUID) {
+        val finnesUnderkjenteSteg = totrinnService.finnesUnderkjenteStegITotrinnsvurdering(behandlingId)
+        var oppgavetype = Oppgavetype.BehandleSak
+        if (finnesUnderkjenteSteg) {
+            oppgavetype = Oppgavetype.BehandleUnderkjentVedtak
+        }
+        with(oppgaveTaskService){
+            ferdigstilleOppgaveTask(behandlingId, oppgavetype)
+            opprettOppgaveTask(behandlingId, Oppgavetype.GodkjenneVedtak)
+        }
```

Comment:
Får du noe mer ut av denne skrivemåten annet enn å bruke fire linjer på å skrive kode som kunne vært skrevet på to?

Reply:
nei, ikke noe mer. Jeg prøvde bare å skrive det med with, men glemte å fjerne det etterpå